azurelinuxagent/ga/update.py

```diff
@@ -275,15 +275,6 @@ def run_latest(self, child_args=None):
                     is_success=False,
                     message=msg)
 
-            if ret is not None and ret > 0:
```

Comment:
should we keep the log statement?

Reply:
it was a repeat of the one above, the only thing relevant here was the blacklisting logic in line 286 but since its not relevant anymore, I removed it